mico-core/src/main/java/io/github/ust/mico/core/dto/PrometheusResponse.java

```diff
@@ -41,7 +40,7 @@
     // TODO: Add comments for fields.
     
     public static final String PROMETHEUS_SUCCESSFUL_RESPONSE = "success";
-    
+
```

Comment:
Check `TODO`on `unpackNested`.

Reply:
Refactoring of this class is part of #427 .